src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/TelephoneCaptureReceiver.java

```diff
@@ -61,6 +61,7 @@ public void receiveMessage(Message<byte[]> message) {
         caze,
         telephoneCapturePayload.getUac(),
         telephoneCapturePayload.getQid(),
+        null,
```

Comment:
Code smell. This means that the changes haven't been applied to Case API... but they will have to be. Telephone capture API will need to know any UAC metadata too